.github/workflows/scheduled_releases.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       matrix:
        # we support the trailing four versions plus the latest pre-release
        # and the upcoming version's pre-release.
-        version: ["1.1.0", "1.2.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0b2", "1.6.0", "1.7.0b1"]
+        version: ["1.2.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0b2", "1.6.0", "1.7.0b1", "1.8.0b1"]
```

Comment:
Should this be 1.7.0rc1 now?

Judging from the comment, should we remove 1.2.0 and 1.6.0b2?

Will this all work with 1.8.0b1 when we're at 1.8.0a1?

Reply:
because we use `~=` for all dbt-* packages it doesn't matter what we use as long as there is a pre-release build tag appended to the version 

> Judging from the comment, should we remove 1.2.0 and 1.6.0b2?

I'd prefer to keep them for now since we have recently introduced changes